RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -337,14 +341,22 @@
 mixedTripletStep = ClassifierMerger.clone()
 mixedTripletStep.inputClassifiers=['mixedTripletStepClassifier1','mixedTripletStepClassifier2']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(mixedTripletStep, mixedTripletStepClassifier1.clone(
+	mva = dict(GBRForestLabel = 'MVASelectorMixedTripletStep_Phase1'),
+	src = 'mixedTripletStepTracks',
+	qualityCuts = [-0.5,0.0,0.5]
+))
+(trackingPhase1 & fastSim).toModify(mixedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
+
+
 from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(mixedTripletStep, TrackLwtnnClassifier.clone(
+trackdnn.toReplaceWith(mixedTripletStep, TrackLwtnnClassifier.clone(
      src = 'mixedTripletStepTracks',
-     qualityCuts = [-0.8, -0.35, 0.1],
+     qualityCuts = [-0.8, -0.35, 0.1]
 ))
-(trackingPhase1 & fastSim).toModify(mixedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
+(trackdnn & fastSim).toModify(mixedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
+
 
 highBetaStar_2018.toReplaceWith(mixedTripletStep, mixedTripletStepClassifier1.clone(
```

Comment:
this and pp_onAA were simply the following in the pre-DNN version
```python
highBetaStar_2018.toModify(mixedTripletStep,qualityCuts = [-0.7,0.0,0.5])
pp_on_AA_2018.toModify(mixedTripletStep, qualityCuts = [-0.5,0.0,0.9])
```

this pattern was used already in other files and we better stay consistent

Reply:
Changed to simpler versions with toModify